airflow/providers/openlineage/plugins/listener.py

```diff
@@ -83,15 +86,16 @@ def on_running():
                 try_number=task_instance.try_number,
             )
 
-            task_metadata = self.extractor_manager.extract_metadata(dagrun, task)
+            with Stats.timer(f"ol.extract.{RunState.RUNNING.value}.{task.task_type}"):
```

Comment:
According to [this docs](https://statsd.readthedocs.io/en/v4.0.1/tags.html), for statsd, metric names must be disaggregated by default because it does not support tags. Let me know if that is a correct approach (it should probably be all lowercase, but is the logic good?)

Reply:
I think that's okay. Metrics users can aggregate on their side.